lib/flutter_code_editor.dart

```diff
@@ -22,6 +22,7 @@ export 'src/folding/foldable_block_type.dart';
 export 'src/folding/invalid_foldable_block.dart';
 export 'src/folding/parsers/highlight.dart';
 
+export 'src/gutter/gutter_style.dart';
 export 'src/line_numbers/line_number_style.dart';
```

Comment:
Rename `line_number_style.dart` to `gutter_style.dart` instead of creating a new `gutter_style.dart`.
Put the deprecated typedef there.